Project.toml

```diff
@@ -12,13 +12,13 @@ AtomsCalculatorsUtilities = "9855a07e-8816-4d1b-ac92-859c17475477"
 Bumper = "8ce10254-0962-460f-a3d8-1f77fea1446e"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 ChunkSplitters = "ae650224-84b6-46f8-82ea-d812ca08434e"
-# EmpiricalPotentials = "38527215-9240-4c91-a638-d4250620c9e2"
 EquivariantModels = "73ee3e68-46fd-466f-9c56-451dc0291ebc"
 ExtXYZ = "352459e4-ddd7-4360-8937-99dcb397b478"
 Folds = "41a02a25-b8f0-4f67-bc48-60067656b558"
 ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
 Interpolations = "a98d9a8b-a2ab-59e6-89dd-64a1c18fca59"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
```

Comment:
why is JSON3 back, I thought we deleted this?

Reply:
Yes, I've used it, but I'll delete it